### DISCUSSION TITLE
How to disable ingress creation?

### DISCUSSION THREAD

**Discussion by srekkas** [bank-vaults/vault-operator#571] [Q&A] [answered]
I create ingress separately, not with operator. How to disable operator creation of ingress?

**Comment by csatib02** [comment, marked as answer]
Hey @srekkas,

Ingress creation is not part of the operator helm-chart, but several CR examples utilises ingresses. Beware that if a CR sets up an ingress it most probably uses it, but of course you can disable them via modifying the CR's yaml before applying.
